tests/tools/pipeline/StringContainerLibrary.cpp

```diff
@@ -117,6 +119,6 @@ static llvm::RegisterPass<ExamplePass> X2("example-pass", "ExamplePass");
 
 char StringContainer::ID;
 
-static RegisterPipe<CopyPipe<StringContainer>> E1(Root);
+static RegisterPipe<CopyPipe<StringContainer>> E1(StringKind);
 static RegisterDefaultConstructibleContainer<StringContainer> C("StringContaine"
                                                                 "r");
```

Comment:
Very sad

Reply:
eh, i mean, i can use "using", but beside that here is not a single character i can drop.

Reply:
`using` in `.cpp` files solves many issues.
Otherwise you can declare the string on another line.